src/util/Rectangle.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Xournal++
  *
- * A rectangle with double precision
+ * A Rectangle<double> with double precision
```

Comment:
The comment here should be about the `Rectangle` class--the new comment is written as if it is a specific instantiation of `Rectangle`.